.github/autoissuelabeler.yml

```diff
@@ -61,3 +61,7 @@
   title: "(?i).*jib.*"
   description: "(?i).*jib.*"
   addcomment: "/cc @geoand"
+- labels: [area/kafka-streams]
+  title: "(?i).*k(afka)?(\s|-)?stream.*"
+  description: "(?i).*k(afka)?(\s|-)?stream.*"
+  addcomment: "/cc @gunnarmorling"
```

Comment:
I wonder if we shouldn't create teams in the Github org and /cc the team instead of the single person. That would be easier when the team member changes

Reply:
Agreed that would be nice. But this PR looks useful anyway.